scripts/run_e2e_backend.sh

```diff
@@ -3,10 +3,17 @@
 jar="./executable/target/executable-0.0.0-SNAPSHOT.jar"
 config="./frontend/cypress/support/backend_config.json"
```

Comment:
@Kadrian fails currently the test
```suggestion
config="./cypress/support/backend_config.json"
```